tasks/DependencyCheck/task/index.ts

```diff
@@ -1,12 +1,15 @@
 import csv from 'csvtojson';
 import { spawnSync } from 'child_process';
 import { LogAnalyticsClient, ILogAnalyticsClient } from './log-analytics';
-import { getVulnData, owaspCheck, cleanDependencyCheckData } from './utility';
+import { downloadVulnData, UploadVulnData, owaspCheck, cleanDependencyCheckData } from './utility';
```

Comment:
casing on UploadVulnData is inconsistent, other function names are camal cased

Reply:
Updated to camelCase